src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents.mdx

```diff
@@ -125,63 +135,69 @@ Repeat these steps for each policy as needed.
+
 <img
-  title="A diagram showing how the By policy issue preference works"
-  alt="A diagram showing how the By policy issue preference works"
+  width="40%;"
+  class="center"
+  title="A diagram showing how the **One issue per policy** option works"
+  alt="A diagram showing how the **One issue per policy** option works"
   src={alertsByPolicy}
 />
 
 <figcaption>
-  When you select the default (**One issue per policy**) as your issue preference, it will group all condition incidents into the same issue.
+  When you select the default option **One issue per policy** as your issue preference, it will group all condition incidents into the same issue.
 </figcaption>
 
-If additional conditions open incidents after the issue opens, the incidents are added to the same issue. Only one issue appears on your **Issues & activity** page for the selected policy.
+## One issue per condition [#preference-condition]
 
-Choose the **One issue per policy** preference if your policy and its conditions have a focused scope (like monitoring one entity). Learn more about [issue preference best practices](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#incident-practices).
+Choose the **One issue per condition** option when you want an individual issue record to focus on a specific condition. If the policy has multiple conditions, you will see separate issue records for each condition on your **Issues & activity** page. Learn more about [your issue creation preferences](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#issue-creation).
 
-## One issue per condition [#preference-condition]
+* If a target breaches a condition for the alert policy, an incident opens and it's created an issue record.
+* If the same target breaches another condition for the alert policy, another incident opens and it's created another issue record.
+* If additional targets open incidents on the same condition after the issue opens, these incidents are added to the issue record already opened for that condition.
 
 <img
-  title="A diagram showing how the By condition issue preference works"
-  alt="A diagram showing how the By condition issue preference works"
+  width="40%;"
+  class="center"
+  title="A diagram showing how the **One issue per condition** option works"
+  alt="A diagram showing how the **One issue per condition** option works"
   src={alertsByCondition}
 />
 
 <figcaption>
-  When you select **One issue per condition** as your issue preference, it will group all incidents by any target for a specific condition into separate issues.
+  When you select the option **One issue per condition** as your issue preference, it will group all incidents by any target for a specific condition into separate issues.
```

Comment:
This is false.

The original description was correct. However, I would suggest using "signal" instead of "target" throughout this document. Switching back and forth between the two terms (which essentially mean the same thing in this context) is confusing.

Reply:
Fixed!